packages/components/src/InputFile/FileTypes.ts

```diff
@@ -0,0 +1,32 @@
+export enum FileTypes {
+  JPEG = "image/jpeg",
+  JPG = "image/jpg",
+  PNG = "image/png",
+  HEIC = "image/heic",
+  PDF = "application/pdf",
+  DOCX = "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+}
+
+export const BASIC_IMAGE_TYPES = ["PNG", "JPEG", "JPG"];
+
+// Converts a MIME type to its readable format
+export const mimeTypeToReadable = (mimeType: string): string => {
+  const entry = Object.entries(FileTypes).find(
+    ([, value]) => value === mimeType,
+  );
+
+  return entry ? entry[0] : mimeType;
+};
+
+// Converts an array of readable file type names to their corresponding MIME types
+export const convertToMimeTypes = (types: string[]): string[] => {
+  return types.map(
+    type => FileTypes[type.toUpperCase() as keyof typeof FileTypes] || type,
+  );
+};
+
+export function formatMimeTypes(types: string[]): string {
+  if (types.length <= 1) return types.join(", ");
```

Comment:
This line works but is a bit weird because you are singling out empty arrays or arrays with one item and then calling join with `", "`  on them which intuitively doesn't make sense but is handled properly in the end by `join`. I would suggest changing it to be explicit about what you want. Something along the lines of:
```
if (types.length === 0) {
    return ""
} else if (types.length === 1) {
    return types[0]
} else {
    return `${types.slice(0, -1).join(", ")} or ${types[types.length - 1]}`;
}
```
This is definitely more verbose but the next person who comes along doesn't need take that extra step to think 🤔 "What does `.join` do with an array with 1 value again?"